db/controllers/app-controllers.js

```diff
@@ -64,3 +65,18 @@ exports.addCommentWithId = (request, response, next) => {
       next(err);
     });
 };
+
+exports.patchArticleVotes = (request, response, next) => {
+  const { article_id } = request.params;
+  const updatedVotes = request.body;
+  const checkArticle = getArticleWithId(article_id);
```

Comment:
We don't need to do this on this endpoint. The reason we do it on the comments one is because there is an overlaps between two situations: empty array but existing article and a non-existent article. Here, we don't need to worry about that. It's not super harmful but if this was a bigger project, an extra call to the db might be seen as wasteful :)